sdk/src/types/block/output/mod.rs

```diff
@@ -73,7 +73,7 @@ pub const OUTPUT_INDEX_RANGE: RangeInclusive<u16> = 0..=OUTPUT_INDEX_MAX; // [0.
 #[derive(Copy, Clone)]
 pub enum OutputBuilderAmount {
     Amount(u64),
-    MinimumStorageDeposit(RentStructure),
+    StorageCost(StorageScoreParameters),
```

Comment:
I would very much appreciate if the variant match the method they are associated to.
`OutputBuilderAmount::Amount` => `new_with_amount`
So I would either do
`OutputBuilderAmount::StorageCost` => `new_with_storage_cost`
or 
`OutputBuilderAmount::MinimumAmount` => `new_with_minimum_amount`
You choose